.github/workflows/setup_ssh.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
+# Sets up ssh keys to allow a ssh connection to localhost. This is needed
+# because localhost is used as remote address to run the tests locally.
 set -ev
 
+mkdir -p ${PWD}/.ssh
```

Comment:
I take it this is necessary because you do `${{ github.workspace }}` in the GHA workflow. But wouldn't it be better to use the temporary file system or `${HOME}` as well there? Or do those options not work?

Reply:
One can also put the key to some tmp folder, but then one has to add each generated key to authorized_keys which fills up the file over time. Since this is a user file that one cannot just be deleted I rather don't do this. So I thought about doing this intermediate solution and put it in the project folder. `${HOME}` would also work, maybe this is even better since a project folder might be deleted.